spec/features/albums_can_be_created_and_viewed_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+feature 'Album' do
+  let!(:user) { create(:user) }
+  let(:test_album_name) { 'My days at Makers' }
+
+  before do
+    visit("/users/sign_in")
+    fill_in 'user_email', with: user.email
+    fill_in 'user_password', with: user.password
+    within(:css, 'div.actions') do
+      click_button 'Log in'
+    end
+  end
+
+  scenario 'View albums' do
```

Comment:
Looks great 🆗 

probably would be helpful to have those test that the user can use those links to navigate to the new album and my albums routes.